src/System.Configuration.ConfigurationManager/tests/System/Configuration/TimeSpanValidatorAttributeTests.cs

```diff
@@ -111,6 +111,7 @@ public void MinValue_Get()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot,"Exception messages are different")]
```

Comment:
Can you just take out the `Assert.Equal(expectedException.Message, result.Message)` in both cases instead? Then the rest of the test runs.

Although, i thought that the exception message would be blank for both cases, so it would be equal anyway,.